### PR TITLE
Install all resalloc-webui directories

### PR DESCRIPTION
**Pull request by schlupov**

### CODE REVIEW

**Comment by praiskup** [comment]
EL7 still shows:
```
    Installed (but unpackaged) file(s) found:
   /usr/share/resallocwebui/static/static/style.css
   /usr/share/resallocwebui/templates/templates/resources.html
Finish: rpmbuild resalloc-4.2.7_g1efba53-1.el7.src.rpm
```

**Comment by schlupov** [comment]
It worked here https://copr.fedorainfracloud.org/coprs/schlupov/Test/build/4359977/

**Comment by praiskup** [comment]
> It worked here https://copr.fedorainfracloud.org/coprs/schlupov/Test/build/4359977/

I bet that the structure of the source RPM is different from what is generated in the reproducible
environment in the CI (`make srpm`).

File: MANIFEST.in [hunk 1]
```diff
@@ -9,3 +9,5 @@ recursive-include shelltests *.sh testrunner testlib
 recursive-include tests *.py
 recursive-include test-tooling resalloc* wrap
 recursive-include build_manpages/build_manpages *.py
+recursive-include resallocwebui/static/*.css
+recursive-include resallocwebui/templates/*.html
```
Comment (praiskup) [inline MANIFEST.in:13]: This seems to be the problem ... note the pattern used in the old MANIFEST lines.

**Comment by praiskup** [comment]
ty

**Pull request merged**